presto-memory/src/main/java/io/prestosql/plugin/memory/data/MemoryTableManager.java

```diff
@@ -404,15 +405,21 @@ private synchronized void spillTable(long id)
         if (!Files.exists(tablePath)) {
             Files.createDirectories(tablePath);
         }
+        ObjectOutputStream oos = null;
         table.setState(Table.TableState.SPILLED);
         try (OutputStream outputStream = Files.newOutputStream(tablePath.resolve(TABLE_METADATA_SUFFIX))) {
-            ObjectOutputStream oos = new ObjectOutputStream(outputStream);
+            oos = new ObjectOutputStream(outputStream);
```

Comment:
it can be written in try () statment itself.

```java
try (OutputStream outputStream = new ObjectOutputStream(Files.newOutputStream(tablePath.resolve(TABLE_METADATA_SUFFIX)))) {
   ...
}
```